docker/elasticsearch/env/docker.env

```diff
@@ -1 +1 @@
-ES_JAVA_OPTS="-Xms256m -Xmx256m -Dlog4j2.formatMsgNoLookups=true"
+ES_JAVA_OPTS="-Xms256m -Xmx256m -Dlog4j2.formatMsgNoLookups=true"
```

Comment:
Though looks good, doesn't seem related to this PR. Can we exclude this file?

Reply:
Can you address the comment?